cats/__init__.py

```diff
@@ -81,25 +83,20 @@ def main(arguments=None):
         if not config:
             print("ERROR: config file not found, exiting now")
             exit(1)
-        now_avg_ci = avg_carbon_intensity(
-            data=CI_forecast, start=datetime.now(), runtime=timedelta(args.duration)
-        )
         estim = greenAlgorithmsCalculator(
             config=config,
             runtime=timedelta(minutes=args.duration),
-            averageBest_carbonIntensity=best_window.value, # TODO replace with real carbon intensity
-            averageNow_carbonIntensity=now_avg_ci,
+            averageBest_carbonIntensity=best_avg.value, # TODO replace with real carbon intensity
+            averageNow_carbonIntensity=now_avg.value,
             **jobinfo,
         ).get_footprint()
 
-        print(" -!-!- Carbon footprint estimation is a work in progress, coming soon!")
-        # Commenting these out while waiting for real carbon intensities
-        # print(f"Estimated emmissions for running job now: {estim.now}")
-        # msg = (
-        #     f"Estimated emmissions for running delayed job: {estim.best})"
-        #     f" (- {estim.savings})"
-        # )
-        # print(msg)
+        print(f"Estimated emmissions for running job now: {estim.now}")
+        msg = (
+            f"Estimated emmissions for running delayed job: {estim.best})"
+            f" (- {estim.savings})"
+        )
+        print(msg)
```

Comment:
My understanding (but I may have got that wrong) was that for now we output everything to sys.stderr since the `at` command takes stdout as input. We need to clean this up in a future PR to be able to output progress updates in stdout, but for now maybe best to output this to stderr?

Reply:
Yes - I think the fix (a different PR for sure) is to wrap the call to `at` inside python rather than having the user use `cats` to generate input to `at` and the command line to pass that through. Doing that portably needs a bit of thought though (and I suspect the idea in my head is not the only sensible design).  See #53

Reply:
Good catch, done.